challenge.js

```diff
@@ -42,7 +42,7 @@
  */
 
 // ANSWER
-
+// I would do var x = document.getElementsByTagName("E");
```

Comment:
E is not the tag name, it's just a name we're using to refer to that element. 

One selector that would grab element E is `p`, since E is a `<p>` element. Another that would work is `div > p`.